core/src/main/scala/filodb.core/SortKeyHelper.scala

```diff
@@ -80,3 +80,24 @@ case class DoubleKeyHelper(segmentLen: Double) extends SingleSortKeyHelper[Doubl
     java.lang.Double.longBitsToDouble(bytes.toLong(true, ByteOrdering.BigEndian))
 }
 
+/**
+ * Right now, you have to specify a prefixLen for the string key helper.
+ * All string keys with identical prefixes (characters 0 to prefixLen - 1) will then be
+ * bucketed into the same segment.
+ * Thus one needs to look through your string sort key data and make sure to bucketize
+ * correctly.
+ *
+ * TODO: perhaps combine prefixLen with a range of chars the last char is allowed to vary on
+ */
+case class StringKeyHelper(prefixLen: Int) extends SingleSortKeyHelper[String] {
```

Comment:
Might be re-usable around the filodb codbase though and belong outside this type.